docs/handbook/systems-administration.md

```diff
@@ -293,6 +293,201 @@ Now user whoever can shutdown the system.
 The `pfexec` command is more flexible in the number of privileges that can be
 assigned to a user.
 
+
+## [Software management](#software-management)
+
+OpenIndiana uses the Image Packaging System, *IPS*, to manage software packages. IPS
+supports installing, updating and searching OpenIndiana software
+repositories. It also allows users to create and publish software packages to
+provide new software for other users. Its features are vast and we only present
+the *essentials* here.
+
+Software packages are *published* to a publisher. An IPS software package
+consists of files, directories, links and dependencies to other packages.
+
+The user interface to install, update, query, generate, etc packages on a
+publisher is `pkg`:
+
+[pkg(1M)](https://illumos.org/man/1M/pkg) is the client for **I**mage
+**P**acking **S**ystem.
```

Comment:
This line seems to have been quite aggressively wrapped.

Reply:
Agreed, but I thought we have a max. line length?  (I can very easily reformat to any max. line length in the coming commits--I've got a few in the pipeline).
These easily fail through tests as we pass these through mkdocs, LaTeX, ... which  are all agnostic to this kind of thing; but agreed we should all be writing to an agreed set of rules.

There is little than can be done via mkdocslint. I'll have a look at the opensolaris style guide which is very good, especially at providing reasons for their rules.

Any technical issues? While I've tried very hard to remove any technically incorrect information, I unfortunately cannot guarantee everything. Is the table overview comprehensible? Of course, this is only if you were to have the time which would be nice.